tests/ttnn/unit_tests/test_reshape.py

```diff
@@ -290,6 +290,7 @@ def test_reshape_tile_layout_only_change_shape(device):
         ((1, 256, 16), (16, 256)),
         ((1, 256, 1024), (1, 256, 16, 64)),
         ((16, 16), (32, 8)),
+        ((32,), (1, 1, 1, 32)),
```

Comment:
could you add non divisible bty32 shapes?

Reply:
Done :)